client/signup/steps/jpo-contact-form/index.jsx

```diff
@@ -49,11 +49,21 @@ const JPOContactFormStep = React.createClass( {
 	renderStepContent() {
 		return (
 			<div className="jpo__contact-form-wrapper">
-				<Card>
-					<ContactUsGraphic />
-					<Button onClick={ this.onSelect }>Add a contact form</Button>
-					<div className="jpo__contact-form-description">Not sure? You can skip this step and add a contact form later.</div>
-				</Card>
+				<a className="jpo-homepage__select-news" href="#" onClick={ this.onSelect }>
+					<Card>
+						<ContactUsGraphic />
+						<Button onClick={ this.onSelect }>
+							{
+								translate( 'Add a contact form' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **46** times:
`translate( 'Add Contact Form' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).